test/e2e/load/load.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2021 ScyllaDB
+
+package load
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx/v2"
+	"github.com/scylladb/gocqlx/v2/table"
+)
+
+type Load struct {
+	session  gocqlx.Session
+	table    *table.Table
+	expected []*test
+}
+
+type test struct {
+	Id   int
+	Data string
+}
+
+func NewLoad(hosts ...string) (*Load, error) {
+	session, err := newSession(hosts...)
+	if err != nil {
+		return nil, err
+	}
+
+	l := &Load{
+		session: session,
+		table: table.New(table.Metadata{
+			Name:    "test.test",
+			Columns: []string{"id", "data"},
+			PartKey: []string{"id"},
+		}),
+		expected: nil,
+	}
+
+	return l, nil
+}
+
+func (l *Load) Close() {
+	l.session.Close()
+}
+
+// Reset closes an existing session and opens a new one.
+// In case an error was returned, Load can no Longer be used.
+func (l *Load) Reset(hosts ...string) error {
+	l.session.Close()
+
+	s, err := newSession(hosts...)
+	if err != nil {
+		return err
+	}
+
+	l.session = s
+	return nil
+}
+
+func (l *Load) Write(n, replicationFactor int) error {
+	if err := l.session.ExecStmt(`DROP KEYSPACE IF EXISTS test`); err != nil {
+		return err
+	}
+
+	if err := l.session.
+		ExecStmt(fmt.Sprintf("CREATE KEYSPACE test WITH replication = {'class' : 'NetworkTopologyStrategy','replication_factor' : %d}", replicationFactor)); err != nil {
```

Comment:
nit: these lines are already pretty long, can we just compare the error on a new line? (applies to other places as well)